support/sier-codec/tests/integration.rs

```diff
@@ -0,0 +1,41 @@
+use sier_codec::Parser;
```

Comment:
maybe a small nuance, as you shared integration tests are a scam, we can rename the "integration test" to "integrated test", this to remind ourselves to use that paradigm (if this is the case for this project) => integrated.rs

Reply:
Good idea. Done.